Imagekit.UnitTests/ImageKitRequestModelValidation.cs

```diff
@@ -147,8 +147,8 @@ public void GetFileListRequest_ModelValidation()
                 Path = "Test",
                 Sort = "Test",
                 SearchQuery = "Test",
-                FileType = "Test"
-               
+                FileType = "Test",
+                Tags = null
```

Comment:
Please remove `Tags=null`. The existing test case should continue to pass. It is a breaking change if user is supposed to pass `Tags=null` now.